code/__HELPERS/mobs.dm

```diff
@@ -231,13 +231,21 @@ GLOBAL_LIST_EMPTY(species_list)
 		checked_health["health"] = health
 	return ..()
 
+///Checks to see if our mob is currently in a do_after with the target as the target
+/mob/proc/is_interacting_with(atom/target)

```

Comment:
Could you turn this into a macro?
It's not like it's being redefined or anything.
Something like `#define IS_INTERACTING_WITH(source, target)`

Reply:
You can delete the proc now that the macro is there.